debian/jellyfin.service

```diff
@@ -6,7 +6,7 @@ After = network.target
 Type = simple
 EnvironmentFile = /etc/default/jellyfin
 User = jellyfin
-ExecStart = /usr/bin/dotnet ${JELLYFIN_EXE} -programdata ${JELLYFIN_DATA} -restartpath ${JELLYFIN_RESTART_SCRIPT} ${JELLYFIN_ADD_OPTS}
+ExecStart = /usr/lib/jellyfin/bin/jellyfin -programdata ${JELLYFIN_DATA} -restartpath ${JELLYFIN_RESTART_SCRIPT} ${JELLYFIN_ADD_OPTS}
```

Comment:
This doesn't seem to work properly since the `jellyfin.dll` has to follow the `jellyfin` binary:

```
$ sudo -u jellyfin /usr/bin/jellyfin -programdata /var/lib/jellyfin
The application to execute does not exist: '/usr/bin/jellyfin.dll'.
```

I think a symlink from `/usr/bin/jellyfin` to `/usr/lib/jellyfin/bin/jellyfin` is a better idea, while keeping the `/usr/bin/jellyfin` paths in the initscripts.